libs/packages/components/src/lib/popover/debounce.decorator.ts

```diff
@@ -0,0 +1,14 @@
+export function debounce(delay: number = 300): MethodDecorator {
```

Comment:
Can you add comments on this for clarity? It seems like it allows a decorator that delays the call of the method it's being decorated over by a given number of milliseconds